### PR TITLE
Replace leapp-pr-welcome-msg-bot with GitHub action

### PR DESCRIPTION
**Pull request by andywaltlova**
For working example see [my test repo](https://github.com/andywaltlova/actions_test_space/pull/14)

(Same [PR in leapp-repository](https://github.com/oamg/leapp-repository/pull/673))

### CODE REVIEW

File: .github/workflows/pr-welcome-msg.yml
Comment (Wecros) [inline .github/workflows/pr-welcome-msg.yml:2]: Small nitpick: should -> there should

**Review by Wecros — CHANGES_REQUESTED** [review]
Otherwise looks good. Good job with linking the private test repo :+1: Will merge after the change.

File: .github/workflows/pr-welcome-msg.yml
Comment (andywaltlova) [reply .github/workflows/pr-welcome-msg.yml]: Fixed

**Review by Wecros — APPROVED** [review]

**Comment by pirat89** [comment]
@Wecros do you plan to merge it?

**Comment by Wecros** [comment]
> @Wecros do you plan to merge it?

Well yes, if I could.

**Comment by Wecros** [comment]
leapp-ci build

**Review by Wecros — DISMISSED** [review]

**Comment by Wecros** [comment]
@pirat89 Should I wait for the `e2e` issues to be fixed or merge it?

**Comment by vojtechsokol** [comment]
Fixed e2e tests on staging passed: https://url.corp.redhat.com/f29fb5a

If it's not a pressing issue I suggest to wait for the fix to land in production (end of the week at the latest), then rerun the tests and merge.

**Review by jmikovic — APPROVED** [review]

**Pull request merged**
